### DISCUSSION TITLE
Helix and Kakoune

### DISCUSSION THREAD

**Discussion by flame-0** [helix-editor/helix#2138] [Q&A] [answered]
- What's the difference between Helix and Kakoune?
- What's something in Helix that isn't present in Kakoune?
- Why Helix over Kakoune, or other text editors?

**Comment by pickfire** [comment, marked as answer]
https://github.com/helix-editor/helix/wiki/Differences-from#kakoune

> What's the difference between Helix and Kakoune?

As far as what I know,

- some keys in helix is still not very consistent yet
- from what I know among the helix developers (or I may be wrong), `v` is more preferable to holding `shift` for extend
- helix takes in some keys from vim which kakoune does not, like `c-u` in insert mode
- helix also have `space xxxx` part of which stolen from doom emacs (or maybe spacemacs)
- helix comes with lsp support and tree-sitter by default, but does not have manual syntax support for each language
- helix does not support plugin unlike kakoune yet

> What's something in Helix that isn't present in Kakoune?

I believe the answers above should have a slight idea of the differences, but helix is still new, there are still changes.

> Why Helix over Kakoune, or other text editors?

Not sure how to answer that, different people have different reasons, part of it for me (and a few others) it just because it is written in rust, also it quite a few stuff that I like being the defaults (don't need to manually setup lsp like kak-lsp).
